lib/db/schemaupdater.go

```diff
@@ -391,12 +391,8 @@ func (db *schemaUpdater) updateSchema6to7(_ int) error {
 		err := t.withNeedLocal(folder, false, func(f FileIntf) bool {
 			name := []byte(f.FileName())
 			global := f.(protocol.FileInfo)
-			gk, delErr = db.keyer.GenerateGlobalVersionKey(gk, folder, name)
-			if delErr != nil {
-				return false
-			}
-			svl, err := t.Get(gk)
-			if err != nil {
+			fl, err := t.getGlobalVL(gk, folder, name)
```

Comment:
VL -> Versions?

It's not that long to require an abberviation, and makes it less cryptic.